model/tensorflow/dffml_model_tensorflow/model/dnn.py

```diff
@@ -187,9 +188,19 @@ def model_dir_path(self, features: Features):
         # 2 classifications whitelist or blacklist
         LOGGER.debug('Loading model with classifications(%d): %r',
                 len(classifications), classifications)
+        param_dict = {}
+        hidden_units=[12, 40, 15]
+        if not self.modelParams is None:
+            for param in self.modelParams:
+                key,val = param.split('=')
+                param_dict[key]=val
+
+            if(param_dict['hidden_units']):
+                hidden_units = ast.literal_eval(param_dict['hidden_units'])
```

Comment:
Niiiiice, good job using `ast`

Reply:
Can you move this to `ParseModelParamsAction` ? You could then do `namespace.model.__dict__.update(param_dict)` and it would set all the parameters which you just parsed.

> I think its `__dict__` but not sure, if you try it and it doesn't work we'll figure it out.

Reply:
Yep. Moved it to ```ParseModelParamsAction```.